### PR TITLE
Update restore-deleted-group.md

### PR DESCRIPTION
**Pull request by Luspin**
Important Change: Removing 'Exchange administrator' permission requirements for O365 group recovery, and keeping GA permissions only.
Remaining changes are just for wording, consistency, etc.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 1]
```diff
@@ -21,19 +21,19 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
+If you're a user who wants to restore an Office 365 group, ask a person in your business with administrative permissions to do these steps for you. For larger businesses, this should mean the internal help desk/technical support team.  
    
-If you've deleted a group, by default it's retained for 30 days. This 30-day period is called "soft-delete" because you can still restore the group. After 30 days, the group and associated content is permanently deleted and cannot be restored.
+If you've deleted a group, it will be retained for 30 days by default. This 30-day period is considered a "soft-delete" because you can still restore the group. After 30 days, the group and its associated contents are permanently deleted and cannot be restored.
   
-During the "soft-delete" period if a user tries to access the site they will get a 404 forbidden message. After this period if the user tries to access the site they will get a 404 not found message.
+During the "soft-delete" period, if a user tries to access the site they will get a 404 _forbidden_ message. After this period if the user tries to access the site they will get a 404 _not found_ message.
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
During the "soft-delete" period, if a user tries to access the site they will get a 404 _forbidden_ message. After this period, if a user tries to access the site, they will get a 404 _not found_ message.
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 2]
```diff
@@ -21,19 +21,19 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
   
 When a group is restored, the following content is restored:
   
-- Azure Active Directory (AD) Office 365 Groups object, properties and members
+- Azure Active Directory (AD) Office 365 groups object, properties and members
     
-- Group SMTP address
+- Group's e-mail addresses
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
- Group's e-mail addresses.
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 3]
```diff
@@ -21,19 +21,19 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
     
-- Exchange Online shared inbox and calendar
+- Exchange Online shared Inbox and calendar
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
- Exchange Online shared Inbox and calendar.
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 4]
```diff
@@ -21,19 +21,19 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
     
 - SharePoint Online team site and files
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
- SharePoint Online team site and files.
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 5]
```diff
@@ -115,11 +115,11 @@ At the message about an untrusted repository, type **Y**. It will take a minute
   
 ### Restore the deleted group
   
-1. Did you install the **AzureADPreview** module, as instructed in the above section "Install the preview version of the Azure Active Directory Module for Windows PowerShell"? Not having the most current **preview** version is the #1 reason these steps don't work for people. 
+1. Did you install the **AzureADPreview** module, as instructed in the above section "Install the preview version of the Azure Active Directory Module for Windows PowerShell"?  Not having the most current **preview** version is the #1 reason these steps don't work for people. 
     
 2. If you haven't already, open a Windows PowerShell window on your computer (it doesn't matter if it's a normal Windows PowerShell window, or one you opened by selecting **Run as administrator**).
     
-3. Run the following commands. Press **Enter** after each command. 
+3. Run the following commands by pressingr **Enter** after each one:
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
3. Run the following commands by pressing **Enter** after each one: 
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 6]
```diff
@@ -187,10 +187,8 @@ Visit the [Microsoft Tech Community](https://techcommunity.microsoft.com/t5/Offi
 
 [Manage Office 365 Groups with PowerShell](https://support.office.com/article/aeb669aa-1770-4537-9de2-a82ac11b0540)
   
-[Delete groups using the Remove-UnifiedGroup cmdlet](https://technet.microsoft.com/en-us/library/mt238270%28v=exchg.160%29.aspx)
+[Delete groups using the Remove-UnifiedGroup cmdlet](https://technet.micro.com/en-us/library/mt238270%28v=exchg.160%29.aspx)
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
[Delete groups using the Remove-UnifiedGroup cmdlet](https://technet.micro.com/library/mt238270%28v=exchg.160%29.aspx).
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 7]
```diff
@@ -21,19 +21,19 @@ description: "Learn how to restore a deleted Office 365 Group using the Exchange
...
-If you've deleted a group, by default it's retained for 30 days. This 30-day period is called "soft-delete" because you can still restore the group. After 30 days, the group and associated content is permanently deleted and cannot be restored.
+If you've deleted a group, it will be retained for 30 days by default. This 30-day period is considered a "soft-delete" because you can still restore the group. After 30 days, the group and its associated contents are permanently deleted and cannot be restored.
   
-During the "soft-delete" period if a user tries to access the site they will get a 404 forbidden message. After this period if the user tries to access the site they will get a 404 not found message.
+During the "soft-delete" period, if a user tries to access the site they will get a 404 _forbidden_ message. After this period if the user tries to access the site they will get a 404 _not found_ message.
   
 When a group is restored, the following content is restored:
   
-- Azure Active Directory (AD) Office 365 Groups object, properties and members
+- Azure Active Directory (AD) Office 365 groups object, properties and members
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
- Azure Active Directory (AD) Office 365 groups object, properties, and members.
```

File: Office365-Admin/create-groups/restore-deleted-group.md [hunk 8]
```diff
@@ -115,11 +115,11 @@ At the message about an untrusted repository, type **Y**. It will take a minute
   
 ### Restore the deleted group
   
-1. Did you install the **AzureADPreview** module, as instructed in the above section "Install the preview version of the Azure Active Directory Module for Windows PowerShell"? Not having the most current **preview** version is the #1 reason these steps don't work for people. 
+1. Did you install the **AzureADPreview** module, as instructed in the above section "Install the preview version of the Azure Active Directory Module for Windows PowerShell"?  Not having the most current **preview** version is the #1 reason these steps don't work for people.
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
1. Did you install the **AzureADPreview** module, as instructed in the previoius section "Install the preview version of the Azure Active Directory Module for Windows PowerShell"?  Not having the most current **preview** version is the #1 reason these steps don't work for people. 
```

**Comment by mypil** [comment]
@Luspin - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

File: Office365-Admin/create-groups/restore-deleted-group.md (hunk 6 above)
Comment (Luspin) [reply Office365-Admin/create-groups/restore-deleted-group.md]: MIssing microsoft.com in the URL here.

**Comment by Luspin** [comment]
Just one missing for @JohanFreelancer9 to check. Thanks

File: Office365-Admin/create-groups/restore-deleted-group.md (hunk 6 above)
Comment (mypil) [inline Office365-Admin/create-groups/restore-deleted-group.md]:
```suggestion
[Delete groups using the Remove-UnifiedGroup cmdlet](https://technet.microsoft.com/library/mt238270%28v=exchg.160%29.aspx)
```

Comment (mypil) [reply Office365-Admin/create-groups/restore-deleted-group.md]: @Luspin - I made the changes in behalf of @JohanFreelancer9. Please check and merge this commit. Thank you.

**Pull request merged**